src/UI/Handlebars/Helpers/Series.js

```diff
@@ -148,24 +148,14 @@ Handlebars.registerHelper('DownloadedStatusColor', function() {
     if (this.downloaded) {
       return "success";
     }
-  //need a check in here to determine if the status exceeds the minavailability specification for the movie
   //should match cases from: src\NzbDrone.Core\Tv\MovieStatusType.cs
-  var x;
-    switch(this.status) {
-	case "announced":
-		    x= 1;
-		    break;
-	case "inCinemas":
-		    x=2;
-		    break;
-	case "released":
-		    x=3;
-		    break;
-    }
-
-  //if (this.status != "released") {
-  var minAvail = parseInt(this.minimumavailability,10);
-  if (x < (isNaN(minAvail) ? 3 : minAvail)) {
+  var MovieStatusType = {
+      tBA : 0, 
+      announced : 1,
+      inCinemas : 2,
+      released : 3
+    };
+  if (MovieStatusType[this.status] < MovieStatusType[this.minimumavailability]) {
```

Comment:
this is the only portion that Im not too sure about or am at a loss as to how to improve it.
Instead of doing select/if statements to handle the cases I just mirrored the enum so that I could do the < check..... not too sure if there is a better way to do this, since the enum is already defined in the C# code.